### PR TITLE
Fixed Android SDK install crash when updating the Tools package

### PR DESCRIPTION
**Pull request by guillaumejenkins**
We no longer update the Tools package as part of Android SDK configuration step. The Tools package already comes installed by default with the Android SDK zip file.

This essentially is a workaround for the current Android SDK crash when updating from Tools 24.3.3 to Tools 24.4 on Windows. This also speeds up the install-reqs process, and helps being future-proof against breaking changes, because we now use a static version of the Tools package.

### CODE REVIEW

File: src/taco-dependency-installer/installers/androidSdkInstaller.ts [hunk 1]
```diff
@@ -37,7 +38,7 @@ class AndroidSdkInstaller extends InstallerBase {
     private static AndroidHomeName: string = "ANDROID_HOME";
     private static AndroidCommand = os.platform() === "win32" ? "android.bat" : "android";
     private static AndroidPackages: string[] = [
-        "tools",
+        // "tools",  // Android SDK comes by default with the tools package, so there is no need to update it. In the future, if we feel we want dependency installer to always install the latest Tools package, then uncomment this.
```
Comment (MSLaguana) [inline src/taco-dependency-installer/installers/androidSdkInstaller.ts]: Maybe add a comment noting that this was causing issues? Is there a known issue in the android installer we can point to?

**Comment by MSLaguana** [comment]
Looks good to me.

**Comment by joshuaskelton** [comment]
Looks good.

**Comment by meenakb** [comment]
:+1:

**Pull request merged**